### ISSUE TITLE
Mailspring Freezes on Launch

### ISSUE THREAD

**Issue by michael-montano** [Foundry376/Mailspring#536]
Mailspring is crashing immediately upon launch. I can still access the menus, but no actions, and though the main window draws, it is completely empty. A screenshot is attached.

##### Are there any related issues?

No.

##### What operating system are you using?
Mac OS 10.13.2

##### What version of Mailspring are you using?
1.0.12

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
No.

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
I have 5 Gmail accounts connected, but nothing else with which to compare.
<img width="1435" alt="screen shot 2018-01-15 at 6 56 32 pm" src="https://user-images.githubusercontent.com/1527383/34969967-75f8c968-fa26-11e7-8f65-b6af2de0f5de.png">

##### Is the issue reproducible with a particular attachment, message, signature, etc?

No.

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
No.

**Comment by michael-montano** [comment]
BTW—Uninstalling and re-installing does not resolve this issue.

**Comment by bengotow** [comment]
Wow this is pretty interesting — I don't think I've seen this before. Can you try launching the app from Terminal by opening Terminal.app and then running `open /Applications/Mailspring.app/Contents/MacOS/Mailspring`? The app will log to the terminal - can you post the output here?

**Comment by hacknug** [comment]
Exact same thing happened on my laptop. Will try to download the app again an see if it still happens.

**Comment by michael-montano** [comment]
Yes. I found the problem cleared when I deleted the app AND the application support folders and then reinstalled.

Michael Montaño
michael.montano@gmail.com (https://link.getmailspring.com/link/local-ef1d2563-4493@Michaels-MacBook-Pro-4.local/0?redirect=mailto%3Amichael.montano%40gmail.com&recipient=reply%2B00174e57a4ae7a9775b884fc2967ba9cba4bb20d7fb9a7ee92cf00000001167ece9092a169ce11364dac%40reply.github.com)(210) 900-4269 (tel:(210)%20900-4269)
110 E. Houston St., Ste. 700, San Antonio, TX 78205 (https://link.getmailspring.com/link/local-ef1d2563-4493@Michaels-MacBook-Pro-4.local/1?redirect=https%3A%2F%2Fmaps.google.com%2F%3Fq%3D110%2520E.%2520Houston%2520St.%252C%2520Ste.%2520700%252C%2520San%2520Antonio%252C%2520TX%252078205&recipient=reply%2B00174e57a4ae7a9775b884fc2967ba9cba4bb20d7fb9a7ee92cf00000001167ece9092a169ce11364dac%40reply.github.com)
